build/setup.d/20-install-packages.sh

```diff
@@ -6,7 +6,7 @@
 pkgs="
 auditd
 build-essential
-docker-engine=1.11.1-0~trusty
+docker-engine=1.12.6-0~ubuntu-trusty
```

Comment:
Ok i'd started the tests on a p2 instance which leads to the use of nvidia-docker. In this case i ran into the symlink issue `libapparmor.so.1 -> libapparmor.so.1.1.0`. With docker 1.12... installed this problem does not exist.

It cost me a lot of time finding this now. Question is what is bad about using a newer minor version even if its already on patch 6